enterprise-suite/alertmanager/alertmanager.yml

```diff
@@ -0,0 +1,55 @@
+# Example Lightbend Console Alertmanager config.
+
+global:
+
+# Routing rules.
+route:
+  receiver: slack
+  # Resend unresolved alerts every 4h.
```

Comment:
I prefer something more generic like "Resend interval for unresolved alerts".  That way if we ever change the value, we don't risk the doco getting out of date.  Just sayin'.